rocAL/rocAL_pybind/examples/rocAL_example_doc/Classification_training_flowerdataset.ipynb

```diff
@@ -0,0 +1,675 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "56c9244e",
+   "metadata": {},
+   "source": [
+    "## Rocal Classification training \n",
+    "In this notebook we will show example of rocAL classification training with small  dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "73bdd89e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd \n",
+    "import torch\n",
+    "from torch.utils.data.dataset import Dataset\n",
+    "from torch.utils.data import DataLoader\n",
+    "from torchvision import transforms, datasets\n",
+    "from torch import nn\n",
+    "import torch.nn.functional as F\n",
+    "import torch.optim as optim\n",
+    "from torch.autograd import Variable\n",
+    "import torchvision.models as models\n",
+    "import time\n",
+    "import math\n",
+    "import tqdm as tqdm\n",
+    "import os\n",
+    "import time \n",
+    "from amd.rocal.plugin.pytorch import ROCALClassificationIterator\n",
+    "from amd.rocal.pipeline import Pipeline\n",
+    "import amd.rocal.fn as fn\n",
+    "import amd.rocal.types as types\n",
+    "from torch.optim import Optimizer\n",
+    "from collections import defaultdict"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3ce165e2",
+   "metadata": {},
+   "source": [
+    "## Setting Dataset path \n",
+    "Here we are setting the dataset path which will be used in the training."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a06c781",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start_time = time.time()\n",
+    "device = torch.device('cpu')\n",
+    "data_dir = './Flower102/split_data/'# Run dataset.py before running the notebook for dataset creation\n",
+    "train_dir = data_dir + '/train'\n",
+    "val_dir = data_dir + '/val'\n",
+    "test_dir = data_dir + '/test'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f3e8acc2",
+   "metadata": {},
+   "source": [
+    "## Defining the Pipeline\n",
+    "We are defining a pipeline for a classification task. Our pipeline will read images from a directory, decode them, apply augmentations and return (image, label) pairs.Our pipeline uses image_random_crop then the output is resized to a dimension of (224,224) followed by normalization."
```

Comment:
Please change our pipeline to "this pipeline" 
Instead of "we are defining" use " the following code defines" (or something similar)

Reply:
Removed the usage of pronouns and instead used generic descriptions for all comments in notebooks